api/src/main/java/uk/gov/dvsa/mot/app/MotrRequestHandler.java

```diff
@@ -113,6 +113,136 @@ public Response getVehicle(@PathParam("registration") String registration,
         }
     }
 
+    @GET
+    @Path("motr/v2/search/commercial/registration/{registration}")
```

Comment:
What is the difference between getVehicle and getCommercialVehicle? Comment with explanation required.

Reply:
Comments added